web/sites/development.services.yml

```diff
@@ -4,10 +4,6 @@
 # 'example.settings.local.php' file, which sits next to this file.
 parameters:
   http.response.debug_cacheability_headers: true
-  twig.config:
```

Comment:
Looks like this snuck in the commit. Let's remove it.

Reply:
done!